tox.ini

```diff
@@ -18,69 +18,69 @@ python =
     3.12: py312
     3.13: py313, lint, docs, pyright
 
-[pytest]
-pythonpath = . tests
-DJANGO_SETTINGS_MODULE = django_minio_storage_tests.settings
-addopts=--tb=short
-python_files = tests.py test_*.py *_tests.py
-django_find_project = false
-
 [testenv]
-commands = pytest {posargs}
-setenv =
-    PYTHONDONTWRITEBYTECODE=1
-    MINIO_STORAGE_ENDPOINT={env:MINIO_STORAGE_ENDPOINT:localhost:9153}
-    MINIO_STORAGE_ACCESS_KEY={env:MINIO_STORAGE_ACCESS_KEY:weak_access_key}
-    MINIO_STORAGE_SECRET_KEY={env:MINIO_STORAGE_SECRET_KEY:weak_secret_key}
-    TOX_ENVNAME={envname}
+package = wheel
```

Comment:
Building wheels to install for tests is significantly faster than building sdists.